ospd_openvas/daemon.py

```diff
@@ -1081,6 +1090,15 @@ def exec_scan(self, scan_id: str):
             dryrun.exec_dry_run_scan(scan_id, self.nvti, OSPD_PARAMS)
             return
 
+        kbdb, err = self.main_db.check_consistency(scan_id)
+        if err < 0:
+            logger.debug(
+                "An old scan with the same scanID was found in the kb. "
+                "Waiting for the kb clean up to finish."
+            )
+            self.stop_scan_cleanup(kbdb, scan_id, kbdb.get_scan_process_id())
+            self.main_db.release_database(kbdb)
```

Comment:
```suggestion
```
database is already cleaned in `stop_scan_cleanup()`

Reply:
Do you mean this lines at end of `stop_scan_cleanup()`?
``` python
 # Clean redis db
            for scan_db in kbdb.get_scan_databases():
                self.main_db.release_database(scan_db)
```
This lines clean the "host's kbs", but not the main kb.
We do the same below, e.g [here](https://github.com/greenbone/ospd-openvas/blob/0772e489128d01f4102b7bdc112c28b3a08ca026/ospd_openvas/daemon.py#L1225)